app/views/settings/dex_connector_oidcs/_form.html.slim

```diff
@@ -0,0 +1,73 @@
+= form_for [:settings, @data_holder], html: { class: "dex-connectors-form", autocomplete: "false" } do |f|
+  / Chrome may autocomplete username into text field before a password field, regardless of name
+  / This *should* prevent that (in combination with 'autocomplete=false' on the form_for; "off" no longer works)
+  / Explanation: Chrome looks for the first password field, while newer versions prefer one named "password"
+  / Firefox behaves similarly.  Hopefully they don't figure out the hidden workaround soon...
+  = f.text_field     :username, style: "display:none", value:"autocomplete trap"
+  = f.password_field :password, style: "display:none", value:"autocomplete bait"
+
+
+  .form-group class="#{error_class_for(@data_holder, :name)}"
+    = f.label :name
+    = f.text_field :name, id: "oidc_name", class: "form-control", value: @data_holder.name, required: true
+    = error_messages_for(@data_holder, :name)
+    small.form-text.text-muted
+      | Name shown to user when selecting a connector
+
+  hr
```

Comment:
I know that adding this creates a larger space between the sections in order to create some context but the `h4` naturally does that.

Reply:
Maybe creating a specific class for the `h4` and increase the margin for that would be better but I guess just removing `hr` is fine. In the ideal world it would be nice to use `fieldset` with `legend` but that will probably require more work which is not the focus of this PR.

Reply:
Removing the `hr` does result in a more compact form, which I generally like.  I also agree that long-term it would be better to use CSS applied to the `h4` rather than the somewhat clunky `hr`s.  However, this change will be inconsistent with the LDAP connector.  Maybe that's ok, since the LDAP connector form is significantly longer.  Removing the other `hr` on Line 41 for consistency also brings the "Client" heading uncomfortably close to the "Basic Auth" toggle.

I'm not sure, then.  Would it be better to remove only this `hr`, to remove both for code consistency, or to retain them and maintain the visual consistency with the LDAP connector form?

Reply:
> Removing the hr does result in a more compact form, which I generally like. I also agree that long-term it would be better to use CSS applied to the h4 rather than the somewhat clunky hrs. However, this change will be inconsistent with the LDAP connector. Maybe that's ok, since the LDAP connector form is significantly longer. 

This is something we should also change on the LDAP connector to keep consistency.

> Removing the other hr on Line 41 for consistency also brings the "Client" heading uncomfortably close to the "Basic Auth" toggle.

I thought I had suggested to add a `br` after the toggle and before the hint text to break the line. Forgot that. :sweat_smile: 

If you do that, it looks fine. See below

![screenshot-20180926183159-168x154](https://user-images.githubusercontent.com/188554/46110607-8be63d80-c1ba-11e8-9035-f272d8dc3434.png)

Reply:
Not sure if it was on purpose but the `br` you added was after the hint text. The hint text is still on the right of the toggle buttons. 

![screenshot-20180926200430-624x82](https://user-images.githubusercontent.com/188554/46114309-70822f00-c1c8-11e8-9059-b328712f24c5.png)

The `br` needs to come above `error_messages_for` line to produce the suggestion on the previous comment. You can remove the `br` you added. Sorry if I wasn't clear enough. :sweat_smile:

Reply:
Oh, whoops.  The `br` I added was to separate the header from the button - moving the text down didn't even occur to me.  Fixed. :)